lib/wallet.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function(options) {
+  var EC = require('elliptic').ec;
+  var ec = new EC('secp256k1');
```

Comment:
It's a little strange to add elliptic here when we could just use that project. But if it's easier for now, it's shouldn't be hard to switch later.